addons/l10n_it_edi/models/account_edi_proxy_user.py

```diff
@@ -1,21 +1,26 @@
-# -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import fields, models
+import logging
+
+from odoo import _, fields, models
+from odoo.exceptions import UserError
+from odoo.addons.account_edi_proxy_client.models.account_edi_proxy_user import AccountEdiProxyError
+from odoo.addons.l10n_it_edi.models.account_move import WAITING_STATES
+
+_logger = logging.getLogger(__name__)
 
 
 class AccountEdiProxyClientUser(models.Model):
     _inherit = 'account_edi_proxy_client.user'
 
-    proxy_type = fields.Selection(
-        selection_add=[('l10n_it_edi', 'Italian EDI')],
-    )
+    proxy_type = fields.Selection(selection_add=[('l10n_it_edi', 'Italian EDI')], ondelete={'l10n_it_edi': 'cascade'})
 
     def _get_proxy_urls(self):
         urls = super()._get_proxy_urls()
         urls['l10n_it_edi'] = {
+            'demo': False,
             'prod': 'https://l10n-it-edi.api.odoo.com',
-            'test': 'https://iap-services-test.odoo.com',
+            'test': 'https://iap-services-test.odoo.com'
```

Comment:
Plz restore the comma here. You should always have one when the dictionary is indented that way.
Then, if you add a new value at the end, you should not touch the previous entry